package.json

```diff
@@ -2,9 +2,11 @@
   "name": "graphql-fields",
   "version": "1.0.2",
   "description": "Turns GraphQLResolveInfo into a map of the requested fields",
-  "main": "index.js",
+  "main": "build/index.js",
   "scripts": {
-    "test": "mocha"
+    "test": "mocha",
+    "build": "babel index.js --out-dir build",
+    "prepublish": "npm run build"
```

Comment:
should this be `prepublishOnly`?

Reply:
@robrichard Oh yes that's better. I'll fix that. Thank you.